header.tpl

```diff
@@ -0,0 +1,13 @@
+ * Copyright {{ YEAR }} NMPolicy Authors.
```

Comment:
I hope that next year, it will not fail when validating the older files.

Reply:
shit... https://github.com/denis-tingaikin/go-header/blob/9edc16b671fc30ca3a30727ae20a474fef305f7c/README.md#bult-in-values

Reply:
Let's just use a regex there.

```
regexp:
    key2: value2 # regexp value just checks regex match. The value should be a valid regexp pattern. Note `key2` should be used in template string as {{ key2 }} or {{ KEY2 }}.
```

Reply:
Based on the documentation, `{{year-range}}` would have done it.

Reply:
Nope, it uses time.Now for the right side of the range.